lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -25,8 +26,8 @@ def self.parse(file)
     end
 
     def self.available?
-      `#{puppet_bin} help strings 2>&1`
-      if $?.success?
+      @raw_json, status = capture2("#{puppet_bin} help strings")
```

Comment:
This shouldn't assign `@raw_json` since this is only checking if the command is present. Also, perhaps make it an array to avoid any potential escaping issues?
```suggestion
      _, status = capture2([puppet_bin, 'help', 'strings'])
```

Other than that this also prints stderr where previously stderr was muted. Perhaps `capture3` should be used and the stderr output discarded?